---
layout: page
permalink: /code/
title: code
description: Standalone programs for data analysis and/or visualization
nav: true
nav_order: 3
---

<div class="page-content">
      <div class="wrapper">
        <div class="post">
  <article class="post-content code clearfix">

<h3>Mutational Distance from Modal Haplotype Calculator</h3>
<a href="https://aemann01.shinyapps.io/mdmhistogram/"><img src="https://github.com/aemann01/aemann01.github.io/blob/master/assets/img/gitMDM.gif?raw=true" height="75%" width="75%" style="padding-bottom: 20px; padding-top: 20px"></a>
<br />
Interactive shiny app that calculates the modal haplotype from Y chromosomal haplotype data and plots the mutational distance of each individual from that modal haplotype.
<br />
<h3>Bubble Tree</h3>
  <a href="https://github.com/aemann01/bubbletree">
    <img src="https://denvercoder1-github-readme-stats.vercel.app/api/pin/?username=aemann01&repo=bubbletree&bg_color=010101&title_color=FFFFFF&hide_border=false&icon_color=B381F1&show_icons=true&text_color=ffffff">
  </a>
<br />

<a href="https://doi.org/10.5281/zenodo.5114948"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.5114948.svg" alt="DOI"></a>
<a href="https://anaconda.org/aemann01/bubbletree"><img src="https://anaconda.org/aemann01/bubbletree/badges/platforms.svg" alt="conda link"></a>
Bubble tree takes frequency table where taxa are rows and samples are columns, a phylogenetic tree, and a sample metadata file to create either a bubble tree or heatmap with abundances plotted on a phylogenetic tree.
<a href="http://github.com/aemann01/bubbleTree"><img src="https://raw.githubusercontent.com/aemann01/bubbleTree/master/img/figexample.png" height="75%" width="75%" style="padding-bottom: 20px; padding-top: 20px"></a>

<h3>GC content by read length plots</h3>
  <a href="https://github.com/aemann01/gcLenCorPlots">
    <img src="https://denvercoder1-github-readme-stats.vercel.app/api/pin/?username=aemann01&repo=gcLenCorPlots&bg_color=010101&title_color=FFFFFF&hide_border=false&icon_color=B381F1&show_icons=true&text_color=ffffff">
  </a>
<br />

<a href="https://zenodo.org/badge/latestdoi/74466977"><img src="https://zenodo.org/badge/74466977.svg" alt="DOI"></a>
Script calculates summary statistics for read length and GC content from a fastq or fasta formatted file and generates a statistics table and plot. Originally written for <a href="https://www.nature.com/articles/s41598-018-28091-9">Mann et al. 2018</a>
<a href="http://github.com/aemann01/gcLenCorPlots"><img src="https://raw.githubusercontent.com/aemann01/gcLenCorPlots/master/img/figexample.png" height="75%" width="75%" style="padding-bottom: 20px; padding-top: 15px"></a>

<h3>Flag PCR and Optical Duplicates</h3>
  <a href="https://github.com/aemann01/flagPCR_opticalDups">
    <img src="https://denvercoder1-github-readme-stats.vercel.app/api/pin/?username=aemann01&repo=flagPCR_opticalDups&bg_color=010101&title_color=FFFFFF&hide_border=false&icon_color=B381F1&show_icons=true&text_color=ffffff">
  </a>
<br />

<a href="https://zenodo.org/badge/latestdoi/74466977"><img src="https://zenodo.org/badge/74466977.svg" alt="DOI"></a>
Script calculates the number of predicted PCR and optical duplicates in a sam file. Returns multiple output files: non-duplicates, PCR duplicates, optical duplicates, and count table (crosstabulation of optical duplicate record groups per sample). Optionally draws figure of location where predicted optical duplicates fall on the tile.


<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
</head>
<body>
<div class="social">
<div class="contact-icons">

</div>

