<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Allison E. Mann </title> <meta name="author" content="Allison E. Mann"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://aemann01.github.io/publications/"> <script src="/assets/js/theme.js?a5ca4084d3b81624bcfa01156dae2b8e"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <div class="navbar-brand social"> <a href="https://aemann01.github.io/assets/pdf/Mann_CV.pdf" title="Download CV"><i class="ai ai-cv" aria-hidden="true" style="font-size:25px;padding-left:5px;padding-right:5px"></i></a> <a href="mailto:%61%6D%61%6E%6E%33@%63%6C%65%6D%73%6F%6E.%65%64%75" title="Contact"><i class="fa fa-envelope" aria-hidden="true" style="font-size:25px;padding-left:5px;padding-right:5px"></i></a> <a href="https://orcid.org/0000-0001-7170-6017" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid ai-3x" style="font-size:25px;padding-left:5px;padding-right:5px"></i></a> <a href="https://scholar.google.com/citations?user=BcHAnkwAAAAJ&amp;hl" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar ai-3x" style="font-size:25px;padding-left:5px;padding-right:5px"></i></a> <a href="https://github.com/aemann01" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa fa-github" style="font-size:25px;padding-left:5px;padding-right:5px"></i></a> <a href="http://www.vprichards-lab.com/" title="Lab Website" rel="external nofollow noopener" target="_blank"><i class="fa fa-flask" aria-hidden="true" style="font-size:25px;padding-left:5px;padding-right:5px"></i></a> <a href="https://www.clemson.edu/science/departments/biosci/directory/profiles/amann3" title="Department Website" rel="external nofollow noopener" target="_blank"><i class="fa fa-briefcase" aria-hidden="true" style="font-size:25px;padding-left:5px;padding-right:5px"></i></a> </div> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/code/">code </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">resources </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/contact">contact</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="https://github.com/rpoCdb/rpoCdatabase/tree/main" rel="external nofollow noopener" target="_blank">rpoCdb</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="https://aemann01.github.io/assets/pdf/bioinformatics_cheat_sheet.pdf">bioinformatics cheat sheet</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="https://christinawarinner.com/outreach/children/adventures-in-archaeological-science/" rel="external nofollow noopener" target="_blank">adventures in archaeological science</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Micro Spectrum</abbr> </div> <div id="Mann2024" class="col-sm-8"> <div class="title">Heterogeneous lineage-specific arginine deiminase expression within dental microbiome species</div> <div class="author"> <em>A.E. Mann</em>, B. Chakraborty , L.M. O’Connell , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'M.M. Nascimento, R.A. Burne, V.P. Richards' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Microbiology Spectrum</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://journals.asm.org/doi/10.1128/spectrum.01445-23" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/mann-et-al-2024-heterogeneous-lineage-specific-arginine-deiminase-expression-within-dental-microbiome-species.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1128/spectrum.01445-23"></span> </div> <div class="abstract hidden"> <p>Arginine catabolism by the bacterial arginine deiminase system (ADS) has anticariogenic properties through the production of ammonia, which modulates the pH of the oral environment. Given the potential protective capacity of the ADS pathway, the exploitation of ADS competent oral microbes through pre- or probiotic applications is a promising therapeutic target to prevent tooth decay. To date, most investigations of the ADS in the oral cavity and its relation to caries have focused on indirect measures of activity, or on specific bacterial groups, yet the pervasiveness and rate of expression of the ADS operon in diverse mixed microbial communities in oral health and disease remains an open question. Here we use a multivariate approach, combining ultra-deep metatranscriptomic sequencing with paired metataxonomic and in vitro citrulline quantification to characterize the microbial community and ADS operon expression in healthy and late-stage cavitated teeth. While ADS activity is higher in healthy teeth, we identify multiple bacterial lineages with upregulated ADS activity on cavitated teeth that are distinct from those found on healthy teeth using both reference-based mapping and denovo assembly methods. Our dual metataxonomic and metatranscriptomic approach demonstrates the importance of species abundance for gene expression data interpretation and that patterns of differential expression can be skewed by low abundance groups. Finally, we identify several potential candidate probiotic bacterial lineages within species that may be useful therapeutic targets for the prevention of tooth decay and propose that the development of a strain-specific, mixed-microbial probiotic may be a beneficial approach given the heterogeneity of taxa identified here across health groups.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Microorg</abbr> </div> <div id="Mancilla2023" class="col-sm-8"> <div class="title">A synthetic formula amino acid diet leads to microbiome dysbiosis, reduced colon length, inflammation, and altered locomoter activity in C57BL/6J mice</div> <div class="author"> V.J. Mancilla , P.N. Braden-Kuhle , K.N. Brice , and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'A.E. Mann, M.T. Williams, Y. Zhang, M.J. Chumley, R.C. Barber, S.N. White, G.W. Boehm, M.S. Allen' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Microorganisms</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.mdpi.com/2076-2607/11/11/2694/htm" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/microorganisms-11-02694.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Although a typical treatment option for metabolic disorders is synthetically derived formula diets, the effects have not been extensively studied, and may render other health consequences. Dietary fiber is essential to protect the gut from opportunistic pathogens and increase the population of commensal bacteria in the colon, which is the most densely populated area for microbiota in the human body. Previous research has demonstrated that diets high in fiber are crucial to human health, as they increase microbial diversity and short chain fatty acid production and allow for proper maintenance of gut tissue. Notably, evidence suggests that high-fiber diets improve cognition and provide neuroprotection, as the gut and brain are connected by the gut-brain axis. Conversely, previous studies in both human subjects and animals have shown that low-fiber diets are associated with decreased gut bacterial diversity, decreased short chain fatty acid production, increased gastrointestinal distress, and increased cognitive dysfunction and anxiety-like behavior. In the current, multidisciplinary study, we examined the effects of a synthetically derived, low-fiber, amino acid diet on anxiety-like behavior, locomotor activity, cognition, the gut microbiome, cytokines, colon length, and short-chain fatty acid production in male C57BL/6J mice. Mice were assigned to one of two diet conditions at post-natal day 21: a standard rodent diet or a synthetic diet analogous to the standard rodent diet, and continued consumption for 13 weeks. The results revealed that this synthetic diet altered locomotor activity. Additionally, by sequencing the 16S rRNA gene from fecal samples (collected weekly) in each group, we identified a decrease in bacterial diversity, and through PICRUSt2 analysis, we predicted changes in metabolic pathway activities. Furthermore, we saw a decrease in the production of short chain fatty acids and a shortening of the colon in mice consuming the synthetic diet. Finally, we measured TNF-α, IL-6, and IL-10 in serum, the hippocampus, and colon, and found that the synthetic diet significantly increased IL-6 production in the hippocampus. These results demonstrate the importance of a multidisciplinary approach to future diet and microbiome studies, as diet not only impacts the gut microbiome, but potentially cognitive and immune function as well.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Micro Spectrum</abbr> </div> <div id="O'Connell2023" class="col-sm-8"> <div class="title">Supragingival mycobiome of HIV-exposed-but-uninfected children reflects a stronger correlation with caries-free associated taxa compared to HIV-infected or uninfected children</div> <div class="author"> L. O’Connell , <em>A.E. Mann</em>, E. Osagie , and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'P. Akhigbe, T. Blouin, A. Soule, O. Obuekwe, A. Omoigberale, R. Burne, M. Coker, V. Richards' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Microbiology Spectrum</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://journals.asm.org/doi/10.1128/spectrum.01491-23" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/o-connell-et-al-2023-supragingival-mycobiome-of-hiv-exposed-but-uninfected-children-reflects-a-stronger-correlation.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Highly active antiretroviral treatment (HAART) has greatly reduced opportunistic infections in HIV-infected individuals. However, even with use of HAART, HIV-infected individuals are still at an increased risk of oral diseases, including dental caries. Dental caries development is associated with microbial community shifts leading to community dysbiosis. HIV infection can significantly alter the bacteriome and mycobiome composition in the oral cavity; however, these impacts have not been assessed for caries initiation and progression. The aim of this study was to characterize the mycobiome for supragingival plaque samples from HIV-infected (HI), HIV-exposed-but-uninfected (HEU), and HIV-unexposed-and-uninfected (HUU) children with and without caries. To accomplish this, ITS1 amplicons of 127 samples were sequenced. We found that HIV infection and exposure resulted in changes to the supragingival plaque mycobiome for both health and caries. Overall, a reduction in community diversity as caries progressed was observed, with HI children having the lowest diversity and HEU children the highest. \emphCandida albicans was the most abundant species identified, with 177 different ASVs. Two \emphC. albicans ASVs dominated the data (53.0% and 38.5% of all taxonomic assignments). The more frequent ASV dominated HI and HUU communities, whereas the second dominated HEU communities. HEU children also had the lowest abundance of \emphC. albicans and the highest number of health-associated taxa.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">µLife</abbr> </div> <div id="Vonaesch2023" class="col-sm-8"> <div class="title">The eukaryome of African children is influenced by geographic location, gut biogeography, and nutritional status.</div> <div class="author"> P. Vonaesch , V. Billy , <em>A.E. Mann</em>, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'E. Morien, A. Habib, J.M. Collard, M. Dédé, N. Kapel, P.J. Sansonetti, L. Parfrey' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>microLife</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://academic.oup.com/microlife/advance-article/doi/10.1093/femsml/uqad033/7227336?searchresult=1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/uqad033.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Eukaryotes have historically been studied as parasites but recent evidence suggests they may be indicators of a healthy gut ecosystem. Here, we describe the eukaryome along the gastrointestinal tract of children 2-5 years and test for associations with clinical factors such as anaemia, intestinal inflammation, chronic undernutrition and age. Children were enrolled from December 2016 - May 2018 in Bangui, Central African Republic and Antananarivo, Madagascar. We analyzed a total of 1,104 samples representing 212 gastric, 187 duodenal, and 705 fecal samples using a metabarcoding approach targeting the full ITS2 region for fungi, and the V4 hypervariable region of the 18S rRNA gene for the overall eukaryome. Roughly half of all fecal samples showed microeukaryotic reads. We find high inter-subject variability, only a handful of taxa that are likely residents of the gastrointestinal tract, and frequent co-occurrence of eukaryotes within an individual. We also find that the eukaryome differs between the stomach, duodenum and feces and is strongly influenced by country of origin. Our dat ashow trends toward higher levels of \emphFusarium equiseti, a mycotoxin producing fungus, and lower levels of the protist \emphBlastocystis in stunted chiuldren compared to non-stunted controls. Overall, the eukaryome is poorly correlated with clinical variables. Our study is of one of the largest cohorts analyzing the human intestinal eukaryome to date and the first to compare the eukaryome across different compartments of the gastrointestinal tract. Our results highlight the importance of studying populations across the world to uncover common features of the eukaryome in health.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/domhain_hiv_thumb-480.webp 480w,/assets/img/publication_preview/domhain_hiv_thumb-800.webp 800w,/assets/img/publication_preview/domhain_hiv_thumb-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/domhain_hiv_thumb.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="domhain_hiv_thumb.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Mann2023a" class="col-sm-8"> <div class="title">Impact of HIV on the Oral Microbiome of Children Living in Sub-Saharan Africa, Determined by Using an \emphrpoC Gene Fragment Metataxonomic Approach.</div> <div class="author"> <em>A.E. Mann</em>, L.M. O’Connell , E. Osagie , and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'P. Akhigbe, O. Obuekwe, A. Omoigberale, C. Kelly, M. Coker, V.P. Richards' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Microbiology Spectrum</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://journals.asm.org/doi/10.1128/spectrum.00871-23" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/spectrum.00871-23.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1128/spectrum.00871-23"></span> <span class="__dimensions_badge_embed__" data-doi="10.1128/spectrum.00871-23" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Children living with HIV have a higher prevalence of oral diseases including caries, the mechanisms of which are not well understood. Here, we test the hypothesis that HIV infection is associated with a more cariogenic oral microbiome, characterized by an increase in bacteria involved in the pathogenesis of caries. We present data generated from supragingival plaques collected from 484 children representing three exposure groups: [1] children living with HIV (HI), [2] children who were perinatally exposed but uninfected (HEU), and [3] unexposed and therefore uninfected children (HUU). We found that the microbiome of HI children is distinct as compared to HEU or HUU children, and that this distinction is more pronounced in diseased teeth as compared to healthy teeth, suggesting that the impact of HIV is more severe as caries progresses. Moreover, we report both an increase in bacterial diversity and decrease in community similarity in our older as compared to younger HI cohort, which may in part be a prolonged effect of HIV and/or its treatment. Finally, while Streptococcus mutans is often a dominant species in late-stage caries, it tends to be found at lower frequency in our HI cohort as compared to other groups. Our results highlight the taxonomic diversity of the supragingival plaque microbiome and suggest that broad and increasingly individualistic ecological shifts are responsible for the pathogenesis of caries in children living with HIV, coupled with a diverse and possibly severe impact on known cariogenic taxa that potentially exacerbates caries.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/diet_preview-480.webp 480w,/assets/img/publication_preview/diet_preview-800.webp 800w,/assets/img/publication_preview/diet_preview-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/diet_preview.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="diet_preview.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Mann2023" class="col-sm-8"> <div class="title">Do I have something in my teeth? The trouble with genetic analyses of diet from archaeological dental calculus</div> <div class="author"> <em>A.E. Mann</em>, J.A. Fellows Yates , Z. Fagernäs , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'R.M. Austin, E.A. Nelson, C.A. Hofman' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Quaternary International</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.sciencedirect.com/science/article/pii/S1040618220307746" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/QI_diet.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.quaint.2020.11.019"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.quaint.2020.11.019" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Dental calculus and other preserved microbiome substrates are an attractive target for dietary reconstruction in past populations through a variety of physical, chemical, and molecular means. Recently, studies have attempted to reconstruct diet from archaeological dental calculus using archaeogenetic techniques. While dental calculus may provide a relatively stable environment for DNA preservation, the detection of plants and animals possibly consumed by an individual through DNA analysis is primarily hindered by microbial richness and incomplete reference databases. Moreover, high genomic similarity within eukaryotic groups - such as mammals - can obfuscate precise taxonomic identification. In the current study we demonstrate the challenges associated with accurate taxonomic identification and authentication of dietary taxa in ancient DNA data using both synthetic and ancient dental calculus datasets. We highlight common errors and sources of contamination across ancient DNA datasets, provide recommendations for dietary DNA validation, and call for caution in the interpretation of diet from dental calculus and other archaeological microbiome substrates.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/necro_preview-480.webp 480w,/assets/img/publication_preview/necro_preview-800.webp 800w,/assets/img/publication_preview/necro_preview-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/necro_preview.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="necro_preview.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Tarone2022" class="col-sm-8"> <div class="title">The Devil is in the Details: Variable impacts of season, BMI, sampling site temperature, and presence of insects on the post-mortem microbiome</div> <div class="author"> A.M. Tarone , <em>A.E. Mann</em>, Y. Zhang , and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'R.R. Zascavage, E.A. Mitchell, E. Morales, T.W. Rusch, M.S. Allen' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2022.1064904/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/fmicb-13-1064904.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2022.1064904 "></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2022.1064904 " data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Post-mortem microbial communities are increasingly investigated as proxy evidence for a variety of factors of interest in forensic science. The reported predictive power of the microbial community to determine aspects of the individual’s post-mortem history (e.g., the post-mortem interval) varies substantially among published research. This observed variation is partially driven by the local environment or the individual themselves. In the current study, we investigated the impact of BMI, sex, insect activity, season, repeat sampling, decomposition time, and temperature on the microbial community sampled from donated human remains in San Marcos, TX using a high-throughput gene-fragment metabarcoding approach. We found that season, temperature at the sampling site, BMI, and sex had a significant effect on the post-mortem microbiome, the presence of insects has a homogenizing influence on the total bacterial community, and that community consistency from repeat sampling decreases as the decomposition process progresses. Moreover, we demonstrate the importance of temperature at the site of sampling on the abundance of important diagnostic taxa. The results of this study suggest that while the bacterial community or specific bacterial species may prove to be useful for forensic applications, a clearer understanding of the mechanisms underpinning microbial decomposition will greatly increase the utility of microbial evidence in forensic casework.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">BMC Oral Health</abbr> </div> <div id="Coker2021" class="col-sm-8"> <div class="title">Dental caries and its association with the oral microbiomes and HIV in young children - Nigeria (DOMHaIN): a cohort study</div> <div class="author"> M.O. Coker , P. Akhigbe , E. Osagie , and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'N.L. Idemudia, O. Igedegbe, N. Chukwumah, R. Adebiyi, A.E. Mann, L.M. O’Connell, O. Obuekwe, A. Omoigberale, M. Charurat, V.P. Richards' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>BMC Oral Health</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://bmcoralhealth.biomedcentral.com/articles/10.1186/s12903-021-01944-y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/Coker2021_Article_DentalCariesAndItsAssociationW.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s12903-021-01944-y"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12903-021-01944-y" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Background: This study seeks to understand better the mechanisms underlying the increased risk of caries in HIV-infected school-aged Nigerian children by examining the relationship between the plaque microbiome and perinatal HIV infection and exposure. We also seek to investigate how perinatal HIV infection and exposure impact tooth-specific microbiomes’ role on caries disease progression. Methods: The participants in this study were children aged 4 to 11 years recruited from the University of Benin Teaching Hospital (UBTH), Nigeria, between May to November 2019. Overall, 568 children were enrolled in three groups: 189 HIV-infected (HI), 189 HIV-exposed but uninfected (HEU) and 190 HIV-unexposed and uninfected (HUU) as controls at visit 1 with a 2.99% and 4.90% attrition rate at visit 2 and visit 3 respectively. Data were obtained with standardized questionnaires. Blood samples was collected for HIV, HBV and HCV screening; CD4, CD8 and Full Blood Count analysis; and plasma samples stored for future investigations; oral samples including saliva, buccal swabs, oropharyngeal swab, tongue swab, dental plaque were collected aseptically from participants at different study visits. Conclusions: Results from the study will provide critical information on how HIV exposure, infection, and treatment, influence the oral microbiome and caries susceptibility in children. By determining the effect on community taxonomic structure and gene expression of dental microbiomes, we will elucidate mechanisms that potentially create a predisposition for developing dental caries. As future plans, the relationship between respiratory tract infections, immune and inflammatory markers with dental caries in perinatal HIV infection and exposure will be investigated.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">PNAS</abbr> </div> <div id="Yates2021" class="col-sm-8"> <div class="title">The evolution and changing ecology of the African hominid oral microbiome</div> <div class="author"> J.A. Fellows Yates , I.M. Velsko , F. Aron , and <span class="more-authors" title="click to view 46 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '46 more authors' ? 'C. Posth, C.A. Hofman, R.M. Austin, C.E. Parker, A.E. Mann, K. Nägele, K. Weedman Arthur, J.W. Arthur, C.C. Bauer, I. Crevecoeur, C. Cupillard, M.C. Curtis, L. Dalén, M. Díaz-Zorita Bonilla, J.C. Díez Fernández-Lomana, D.G. Drucker, E. Escribano Escrivá, M. Francken, V.E. Gibbon, M. Gonzalez Morales, A. Grande Mateu, K. Harvati, A.G. Henry, L. Humphrey, M. Menéndez, D. Mihailović, M. Peresani, S. Rodríguez Moroder, M. Roksandic, H. Rougier, S. Sázelová, J.T. Stock, Guy Straus L., J. Svoboda, B. Teßmann, M.J. Walker, R.C. Power, C.M. Lewis, K. Sankaranarayanan, K. Guschanski, R. Wrangham, F.E. Dewhirst, D.C. Salazar-Garcia, J. Krause, A. Herbig, C. Warinner' : '46 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">46 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.pnas.org/content/118/20/e2021655118" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/e2021655118.full.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1073/pnas.2021655118"></span> <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2021655118" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The oral microbiome plays key roles in human biology, health, and disease, but little is known about the global diversity, variation, or evolution of this microbial community. To better understand the evolution and changing ecology of the human oral microbiome, we analysed 124 dental biofilm metagenomes from humans, including Neanderthals and Late Pleistocene to present-day modern humans, chimpanzees, and gorillas, as well as New World howler monkeys for comparison. We find that a core microbiome of primarily biofilm-structural taxa has been maintained throughout African hominid evolution, and these microbial groups are also shared with howler monkeys, suggesting that they have been important oral members since before the catarrhine-platyrrhine split ca. 40 million years ago. However, community structure and individual microbial phylogenies do not closely reflect host relationships, and the dental biofilms of Homoand chimpanzees are distinguished by major taxonomic and functional differences. Reconstructing oral metagenomes up to 100 thousand years ago, we show that the microbial profiles of both Neanderthals and modern humans are highly similar, sharing functional adaptations in nutrient metabolism. These include an apparent Homo-specific acquisition of salivary amylase-binding capability by oral streptococci, suggesting microbial co-adaptation with host diet. We additionally find evidence of shared genetic diversity in the oral bacteria of Neanderthal and Upper Palaeolithic modern humans that is not observed in later modern human populations. Differences in the oral microbiomes of African hominids provide insights into human evolution, the ancestral state of the human microbiome, and a temporal framework for understanding microbial health and disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Microorg.</abbr> </div> <div id="Mancilla2021" class="col-sm-8"> <div class="title">The adult phenylketonuria (PKU) gut microbiome</div> <div class="author"> V.J. Mancilla , <em>A.E. Mann</em>, Y. Zhang , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'M.S. Allen' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Microorganisms</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.mdpi.com/2076-2607/9/3/530" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/microorganisms-09-00530-v2.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3390/microorganisms9030530"></span> <span class="__dimensions_badge_embed__" data-doi="10.3390/microorganisms9030530" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Phenylketonuria (PKU) is an inborn error of phenylalanine metabolism primarily treatedthrough a phenylalanine-restrictive diet that is frequently supplemented with an amino acid formulato maintain proper nutrition. Little is known of the effects of these dietary interventions on thegut microbiome of PKU patients, particularly in adults. In this study, we sequenced the V4 regionof the 16S rRNA gene from stool samples collected from adults with PKU (n= 11) and non-PKUcontrols (n= 21). Gut bacterial communities were characterized through measurements of diversityand taxa abundance. Additionally, metabolic imputation was performed based on detected bacteria.Gut community diversity was lower in PKU individuals, though this effect was only statisticallysuggestive. A total of 65 genera across 5 phyla were statistically differentially abundant betweenPKU and control samples (p&lt; 0.001). Additionally, we identified six metabolic pathways that differedbetween groups (p&lt; 0.05), with four enriched in PKU samples and two in controls. While the childPKU gut microbiome has been previously investigated, this is the first study to explore the gutmicrobiome of adult PKU patients. We find that microbial diversity in PKU children differs from PKUadults and highlights the need for further studies to understand the effects of dietary restrictions.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/triatome_preview-480.webp 480w,/assets/img/publication_preview/triatome_preview-800.webp 800w,/assets/img/publication_preview/triatome_preview-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/triatome_preview.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="triatome_preview.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Mann2020" class="col-sm-8"> <div class="title">Comparison of the Bacterial Gut Microbiome of North American Trypanosoma spp. With and Without Trypanosoma cruzi</div> <div class="author"> <em>A.E. Mann</em>, E.A. Mitchell , Y. Zhang , and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'R. Curtis-Robles, S. Thapa, S.A. Hamer, M.S. Allen' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.00364/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/fmicb-11-00364-1.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2020.00364"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2020.00364" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Chagas disease, caused by the hemoflagellate protist Trypanosoma cruzi, affects nearly 6 million people worldwide, mainly in Latin America. Hematophagous triatomine insects (“kissing bugs”) are the primary vectors of T. cruzi throughout the Americas and feed on a variety of animals, including humans. Control of triatomines is central to the control of T. cruzi infection. Recent advances in mitigation of other insect-borne diseases via the manipulation of insect-associated bacteria as a way to halt or slow disease transmission has opened questions to the applicability of these methods to Chagas disease vectors. Few studies have examined the hindgut microbiome of triatomines found in North America. In the current study, two species of triatomines were collected across Texas, United States, screened for the presence of T. cruzi, and analyzed for the bacterial composition of their hindguts using a 16S rRNA gene-fragment metabarcoding approach. We compared diversity of microbial community profiles across 74 triatomine insects to address the hypothesis that the richness and abundance of bacterial groups differ by T. cruzi infection and strain type, blood meal engorgement status, insect species, sex, and collection location. The gut microbial community of individual triatomines was characterized by low intraindividual taxonomic diversity and high interindividual variation that was weakly predicted by triatomine species, and was not predicted by triatomine sex, collection location, T. cruzi infection status, or blood meal score. However, we did find bacterial groups enriched in T. cruzi-positive individuals, including Enterobacterales, and Petrimonas. Additionally, we detected Salmonella enterica subspecies diarizonae in three triatomine individuals; this species is commonly associated with reptiles and domesticated animals and is a pathogen of humans. These data suggest that Triatoma spp. in Texas have variable patterns of colonized and transient bacteria, and may aid in development of novel means to interfere with transmission of the Chagas disease parasite T. cruzi. Deeper understanding of the effects of parasite infection on diverse insect vector microbiomes may highlight disease transmission risk and facilitate discovery of possible intervention strategies for biological control of this emerging vector-borne disease of global health significance.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/primateeuk_preview-480.webp 480w,/assets/img/publication_preview/primateeuk_preview-800.webp 800w,/assets/img/publication_preview/primateeuk_preview-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/primateeuk_preview.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="primateeuk_preview.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Mann2021" class="col-sm-8"> <div class="title">Biodiversity of protists and nematodes in the wild non-human primate gut</div> <div class="author"> <em>A.E. Mann</em>, F. Mazel , M.A. Lemay , and <span class="more-authors" title="click to view 18 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '18 more authors' ? 'E. Morien, V. Billy, M. Kowalewski, A. Di Fiore, A. Link, T.L. Goldberg, S. Tecot, A.L. Baden, A. Gomez, M.L. Sauther, F.P. Cuozzo, G.A.O. Rice, N.J. Dominy, R. Stumpf, R.J. Lewis, L. Swedell, K. Amato, L. Parfrey' : '18 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">18 more authors</span> </div> <div class="periodical"> <em>The ISME Journal</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41396-019-0551-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/s41396-019-0551-4.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41396-019-0551-4"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41396-019-0551-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Documenting the natural diversity of eukaryotic organisms in the nonhuman primate (NHP) gut is important for understanding the evolution of the mammalian gut microbiome, its role in digestion, health and disease, and the consequences of anthropogenic change on primate biology and conservation. Despite the ecological significance of gut-associated eukaryotes, little is known about the factors that influence their assembly and diversity in mammals. In this study, we used an 18S rRNA gene fragment metabarcoding approach to assess the eukaryotic assemblage of 62 individuals representing 16 NHP species. We find that cercopithecoids, and especially the cercopithecines, have substantially higher alpha diversity than other NHP groups. Gut-associated protists and nematodes are widespread among NHPs, consistent with their ancient association with NHP hosts. However, we do not find a consistent signal of phylosymbiosis or host-species specificity. Rather, gut eukaryotes are only weakly structured by primate phylogeny with minimal signal from diet, in contrast to previous reports of NHP gut bacteria. The results of this study indicate that gut-associated eukaryotes offer different information than gut-associated bacteria and add to our understanding of the structure of the gut microbiome.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">AJPA</abbr> </div> <div id="Ziesemer2019" class="col-sm-8"> <div class="title">The efficacy of whole genome capture on ancient dental calculus and dentin</div> <div class="author"> K.A. Ziesemer , J. Ramos-Madrigal , <em>A.E. Mann</em>, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'B.W. Brandt, K. Sankaranarayanan, A.T. Ozga, M. Hoogland, C.A. Hofman, D.C. Salazar-Garcia, B. Frohlich, G.R. Milner, A.C. Stone, M. Aldenderfer, C.M. Lewis, C.L. Hofman, C. Warinner, H. Schroeder' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>American Journal of Physical Anthropology</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajpa.23763" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/ajpa.23763.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1002/ajpa.23763"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/ajpa.23763" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Dental calculus is among the richest known sources of ancient DNA in the archaeological record. Although most DNA within calculus is microbial, it has been shown to contain sufficient human DNA for the targeted retrieval of whole mitochondrial genomes. Here, we explore whether calculus is also a viable substrate for whole human genome recovery using targeted enrichment techniques. Total DNA extracted from 24 paired archaeological human dentin and calculus samples was subjected to whole human genome enrichment using in‐solution hybridization capture and high‐throughput sequencing. Total DNA from calculus exceeded that of dentin in all cases, and although the proportion of human DNA was generally lower in calculus, the absolute human DNA content of calculus and dentin was not significantly different. Whole genome enrichment resulted in up to four‐fold enrichment of the human endogenous DNA content for both dentin and dental calculus libraries, albeit with some loss in complexity. Recovering more on‐target reads for the same sequencing effort generally improved the quality of downstream analyses, such as sex and ancestry estimation. For nonhuman DNA, comparison of phylum‐level microbial community structure revealed few differences between precapture and postcapture libraries, indicating that off‐target sequences in human genome‐enriched calculus libraries may still be useful for oral microbiome reconstruction. While ancient human dental calculus does contain endogenous human DNA sequences, their relative proportion is low when compared with other skeletal tissues. Whole genome enrichment can help increase the proportion of recovered human reads, but in this instance enrichment efficiency was relatively low when compared with other forms of capture. We conclude that further optimization is necessary before the method can be routinely applied to archaeological samples.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/dvc_preview-480.webp 480w,/assets/img/publication_preview/dvc_preview-800.webp 800w,/assets/img/publication_preview/dvc_preview-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/dvc_preview.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="dvc_preview.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Mann2018" class="col-sm-8"> <div class="title">Differential preservation of endogenous human and microbial DNA in dental calculus and dentin</div> <div class="author"> <em>A.E. Mann</em>, S. Sabin , K.A. Ziesemer , and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Å. Vågene, H. Schroeder, A. Ozga, K. Sankaranarayanan, C.A. Hofman, J.A. Fellows Yates, D.C. Salazar-Garcia, B. Frohlich, M. Aldenderfer, M. Hoogland, C. Read, J. Krause, C. Hofman, K. Bos, C. Warinner' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>Scientific Reports</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41598-018-28091-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/s41598-018-28091-9.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41598-018-28091-9"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-018-28091-9" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Dental calculus (calcified dental plaque) is prevalent in archaeological skeletal collections and is a rich source of oral microbiome and host-derived ancient biomolecules. Recently, it has been proposed that dental calculus may provide a more robust environment for DNA preservation than other skeletal remains, but this has not been systematically tested. In this study, shotgun-sequenced data from paired dental calculus and dentin samples from 48 globally distributed individuals are compared using a metagenomic approach. Overall, we find DNA from dental calculus is consistently more abundant and less contaminated than DNA from dentin. The majority of DNA in dental calculus is microbial and originates from the oral microbiome; however, a small but consistent proportion of DNA (mean 0.08 ± 0.08%, range 0.007–0.47%) derives from the host genome. Host DNA content within dentin is variable (mean 13.70 ± 18.62%, range 0.003–70.14%), and for a subset of dentin samples (15.21%), oral bacteria contribute &gt; 20% of total DNA. Human DNA in dental calculus is highly fragmented, and is consistently shorter than both microbial DNA in dental calculus and human DNA in paired dentin samples. Finally, we find that microbial DNA fragmentation patterns are associated with guanine-cytosine (GC) content, but not aspects of cellular structure.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/spaam_preview-480.webp 480w,/assets/img/publication_preview/spaam_preview-800.webp 800w,/assets/img/publication_preview/spaam_preview-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/spaam_preview.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="spaam_preview.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Warinner2017" class="col-sm-8"> <div class="title">A robust framework for microbial archaeology</div> <div class="author"> C. Warinner , A. Herbig , <em>A.E. Mann</em>, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'J.A. Fellows Yates, C.L. Weiss, H.A. Burbano, L. Orlando, J. Krause' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Annual Review of Genomics and Human Genetics</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.annualreviews.org/doi/full/10.1146/annurev-genom-091416-035526" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/annurev-genom-091416-035526.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1146/annurev-genom-091416-035526"></span> <span class="__dimensions_badge_embed__" data-doi="10.1146/annurev-genom-091416-035526" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Microbial archaeology is flourishing in the era of high-throughput sequencing, revealing the agents behind devastating historical plagues, identifying the cryptic movements of pathogens in prehistory, and reconstructing the ancestral microbiota of humans. Here, we introduce the fundamental concepts and theoretical framework of the discipline, then discuss applied methodologies for pathogen identification and microbiome characterization from archaeological samples. We give special attention to the process of identifying, validating, and authenticating ancient microbes using high-throughput DNA sequencing data. Finally, we outline standards and precautions to guide future research in the field.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Sci. Rep.</abbr> </div> <div id="Ziesemer2015" class="col-sm-8"> <div class="title">Intrinsic challenges in ancient microbiome reconstruction using 16S rRNA gene amplification</div> <div class="author"> K.A. Ziesemer , <em>A.E. Mann</em>, K. Sankaranarayanan , and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'H. Schroeder, A. Ozga, B.W. Brandt, E. Zaura, A. Waters-Rist, M. Hoogland, D.C. Salazar-Garcia, M. Aldenderfer, C. Speller, J. Hendy, D.A. Weston, S.J. MacDonald, G.H. Thomas, M.J. Collins, C.M. Lewis, C. Hofman, C. Warinner' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>Scientific Reports</em>, 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/srep16498" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/srep16498.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/srep16498"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/srep16498" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>To date, characterization of ancient oral (dental calculus) and gut (coprolite) microbiota has been primarily accomplished through a metataxonomic approach involving targeted amplification of one or more variable regions in the 16S rRNA gene. Specifically, the V3 region (E. coli 341–534) of this gene has been suggested as an excellent candidate for ancient DNA amplification and microbial community reconstruction. However, in practice this metataxonomic approach often produces highly skewed taxonomic frequency data. In this study, we use non-targeted (shotgun metagenomics) sequencing methods to better understand skewed microbial profiles observed in four ancient dental calculus specimens previously analyzed by amplicon sequencing. Through comparisons of microbial taxonomic counts from paired amplicon (V3 U341F/534R) and shotgun sequencing datasets, we demonstrate that extensive length polymorphisms in the V3 region are a consistent and major cause of differential amplification leading to taxonomic bias in ancient microbiome reconstructions based on amplicon sequencing. We conclude that systematic amplification bias confounds attempts to accurately reconstruct microbiome taxonomic profiles from 16S rRNA V3 amplicon data generated using universal primers. Because in silico analysis indicates that alternative 16S rRNA hypervariable regions will present similar challenges, we advocate for the use of a shotgun metagenomics approach in ancient microbiome reconstructions.</p> </div> </div> </div> </li></ol> </div> <meta name="viewport" content="width=device-width, initial-scale=1"> <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> <div class="social"> <div class="contact-icons"> </div> </div> </article> </div> </div> <footer class="sticky-bottom mt-5" role="contentinfo"> <div class="container"> © Copyright 2024 Allison E. Mann. Powered by <a href="http://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?fcfacfb8c6281f5e68d5a7d348186eb1"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>